src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/EnvelopeRepository.java

```diff
@@ -8,10 +8,10 @@
 public interface EnvelopeRepository extends JpaRepository<Envelope, UUID> {
 
     /**
-     * Finds envelopes for a given jurisdiction.
+     * Finds envelopes with processed status for a given jurisdiction.
```

Comment:
The query does not state whether status is processed or not - can be applied to any 😉